services/authoring/src/utils/hook.ts

```diff
@@ -3,4 +3,6 @@ export default interface Hook {
   keyPath: string;
   type: string;
   url: string;
+  tags?: string[];
```

Comment:
Do we have tags?

Reply:
(Guess, we can still add it as It's just the api/storage layer)

Reply:
It's coming Soon 🔜